Library/Formula/agda.rb

```diff
@@ -101,11 +104,13 @@ module simple-test where
       snoc [] x = x :: []
       snoc (x :: xs) y = x :: (snoc xs y)
     EOS
-    system bin/"agda", "-c", "--no-main", "--safe", test_file_path
     system bin/"agda", "--js", "--safe", test_file_path
+    if which("ghc")
```

Comment:
By using `which`, I’m hoping to cover the case where the user has GHC installed using other methods than Homebrew.